docs-rtd/howto/manage-migrations.md

```diff
@@ -0,0 +1,101 @@
+(manage-migrations)=
+
+# Manage migrations
+
+The following document shows you what changes, if any, are needed to your Terraform plan when migrating a model(s) between Juju controllers and/or JAAS.
+
+## Migrating to another Juju controller
+
+When a model is migrated to a new Juju controller, no changes are needed. Simply update you plan to connect to the new controller.
+
+The only exception to this scenario is when cross-model relations are involved. If both models involved in a cross-model relation are moved, no changes are necessary. If only one model involved in the relation is moved, see below.
+
+(manage-migration-cross-controller-relations)=
+
+### Cross-controller relations
+
+When a model consumes an offer from another model, this is known as a cross-model relation.
```

Comment:
This shouldn't be defined here. TFJ relies on Juju docs for notions that are ultimately owned by the Juju controller.

Reply:
Removed.